ui/partials/perf/comparectrl.html

```diff
@@ -10,8 +10,7 @@
     <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
       <h1>Perfherder Compare Revisions</h1>
       <ul class="list-inline push-information">
```

Comment:
Could you put the `<ul ....>` element inside the directive? It's repeated across comparectrl and comparesubtestctrl.